content/docs/preferences.md

```diff
@@ -467,3 +450,66 @@ If you prefer to edit XML instead of using the GUI to modify shortcuts, you may
+
+In general, if you want the glyph to stay the same and change the bytes on the disk, then use the `Convert to...` entries; whereas if the glyphs shown don't match what you think the bytes of the data should represent, you probably need to use one of the upper entries to change the interpretation of the bytes.
+
+#### Language Menu
+
+This menu shows the active syntax highlighter lexer (including [User Defined Languages](../user-defined-language-system/)), and allows you to change the syntax highlighter for the current file.  The syntax highlighting colors are set in the [Style Configurator](#style-configurator) or in the [UDL dialog](../user-defined-language-system/#udl-dialog-box-or-window).
+
+### Preferences for Advanced Users
+
+The following settings are for rather specific needs and could cause some confusion if they are enabled. As a result they are not set via UI but in `config.xml`. Note that you should close Notepad++ then edit `config.xml` following the best practices in the [Editing Configuration Files](../config-files/#editing-configuration-files) section to prevent your modification from being erased or overwritten when Notepad++ exits.
+
+* Allow regex backward search: Backward regex searching is foribidden by default (starting in v7.8.7) due to sometimes surprising results. However, if this feature is needed, you can set `regexBackward4PowerUser` attribute to `yes` in the `FindHistory` tag of `config.xml` to eanable this option:
+```
+<FindHistory nbMaxFindHistoryPath="10" nbMaxFindHistoryFilter="10" nbMaxFindHistoryFind="10" nbMaxFindHistoryReplace="10" matchWord="no" matchCase="no" wrap="yes" directionDown="yes" fifRecuisive="yes" fifInHiddenFolder="no" fifFilterFollowsDoc="no" fifFolderFollowsDoc="no" searchMode="0" transparencyMode="1" transparency="150" dotMatchesNewline="no" isSearch2ButtonsMode="yes" "regexBackward4PowerUser"="yes">
+```
+Simply add `"regexBackward4PowerUser"="yes"` if this option is absent.
+
+* Changing the command-line interpreter used: by default, **File > Open Containing Folder > cmd** will launch the `cmd.exe` command-line interpreter.  If you have a preferred command-line interpreter (such as `powershell`), you can add another `<GUIConfig...>` tag inside the `<GUIConfigs>` section:
+```
+<GUIConfig name="commandLineInterpreter">powershell</GUIConfig>
+```
+If your command-line interpreter is not in your path, make sure to include the drive and folder in the contents of that tag.  If there are spaces, make sure to use quotes around the path:
+```
+<GUIConfig name="commandLineInterpreter">"c:\path\with spaces\to\cli.exe"</GUIConfig>
+```
```

Comment:
There is a new "hidden" configuration setting, the enableSelectForegroundColour.xml file.
As the name suggests, its existence allows to change the foreground color of the selected line.
See https://github.com/notepad-plus-plus/notepad-plus-plus/pull/9817 for more information and the reason why it cannot be part of the user interface.

Reply:
@Ekopalypse , it's already mentioned in the Global Styles entry on "Selected Text Colour":
![image](https://user-images.githubusercontent.com/17455758/118869542-6130e200-b89a-11eb-921d-4d23377a1c1c.png)

... and in the Config Files document.